vnext/.flowconfig

```diff
@@ -95,8 +93,10 @@ module.file_ext=.windows.js
 
 munge_underscores=true
 
-module.name_mapper='^react-native$' -> '<PROJECT_ROOT>/index.js'
+module.name_mapper='^react-native$' -> '<PROJECT_ROOT>/index.windows.js'
+module.name_mapper='^react-native-windows$' -> '<PROJECT_ROOT>/index.windows.js'
 module.name_mapper='^react-native/\(.*\)$' -> '<PROJECT_ROOT>/\1'
+module.name_mapper='^react-native-windows/\(.*\)$' -> '<PROJECT_ROOT>/\1'
```

Comment:
>.* [](start = 44, length = 2)

[nit] requires backtracking, potentially unexpected results if multiple ')'s

Reply:
?

Reply:
Never mind, it's nitpick irrelevant in this context. As specified, "rnw/(a))" would map to "<PRJROOT>/a)"; a  sub-pattern like "\([^\(\)]*\)" would map to "<PRJROOT>/a". 

---
In reply to: [453108200](https://github.com/microsoft/react-native-windows/pull/5508#discussion_r453108200) [](ancestors = 453108200)